<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRoundFinished</class>
 <widget class="QDialog" name="DlgRoundFinished">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Round finished</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="laTitle">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select which documents to print now:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnResults">
       <property name="text">
        <string>Results and next matches</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStandings">
       <property name="text">
        <string>Current standings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBracket">
       <property name="text">
        <string>Bracket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMatrix">
       <property name="text">
        <string>Match matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInOut">
       <property name="text">
        <string>In/out list</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnOkay">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnOkay</tabstop>
  <tabstop>btnResults</tabstop>
  <tabstop>btnStandings</tabstop>
  <tabstop>btnBracket</tabstop>
  <tabstop>btnMatrix</tabstop>
  <tabstop>btnInOut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOkay</sender>
   <signal>clicked()</signal>
   <receiver>DlgRoundFinished</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
